src/tests/ftest/object/create_many_dkeys.py

```diff
@@ -61,22 +64,17 @@ def write_a_bunch_of_values(self, how_many):
             val = ioreq.single_fetch(c_dkey, c_akey, len(the_data) + 1)
             exp_value = val.value.decode("utf-8")
             if the_data != exp_value:
-                self.fail("ERROR: Data mismatch for dkey = {0}, akey={1}, "
-                          "Expected Value={2} and Received Value={3}\n"
-                          .format("dkey {0}".format(key),
-                                  "akey {0}".format(key),
-                                  the_data,
-                                  exp_value))
+                self.log.debug("Expected Value: %s", the_data)
+                self.log.debug("Received Value: %s", exp_value)
+                self.fail("ERROR: Data mismatch for dkey='dkey {key}', akey='akey {key}'")
 
             if key > last_key:
-                print("veried: {}".format(key))
+                print("verified: {}".format(key))
                 sys.stdout.flush()
                 last_key = key + inc
 
-        print("starting destroy")
-        self.container.close()
-        self.container.destroy()
-        print("destroy complete")
+        self.log_step("Destroying the container")
+        container.destroy()
```

Comment:
I thought teardown will handle the container destroy... Why we need to destroy the container in the test case?

Reply:
The `write_a_bunch_of_values()` method that executes these lines also creates the container and is called a couple times - it is part of the test steps.